File: android/src/main/java/com/getcapacitor/community/admob/models/RewardAdPluginEvents.kt
package com.getcapacitor.community.admob.models

enum class RewardAdPluginEvents(val webEventName: String) {
    Loaded("onRewardedVideoAdLoaded"),
    FailedToLoad("onRewardedVideoAdFailedToLoad"),
    Showed("onRewardedVideoAdShowed"),
    FailedToShow("onRewardedVideoAdFailedToShow"),
    Dismissed("onRewardedVideoAdDismissed"),
    Rewarded("onRewardedVideoAdReward")
}

File: ios/Sources/AdMobPlugin/Rewarded/RewardAdPluginEvents.swift
public enum RewardAdPluginEvents: String {
    case Loaded = "onRewardedVideoAdLoaded"
    case FailedToLoad = "onRewardedVideoAdFailedToLoad"
    case Showed = "onRewardedVideoAdShowed"
    case FailedToShow = "onRewardedVideoAdFailedToShow"
    case Dismissed = "onRewardedVideoAdDismissed"
    case Rewarded = "onRewardedVideoAdReward"
}
